src/cpp/desktop/html/connect.html

```diff
@@ -0,0 +1,89 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
```

Comment:
For WCAG compliance, a page must have a `<title>`in `<head>`. Recommend including one with same contents as  the `h1`. Same in error.html.

Reply:
Done in both places.